simulation/sub8_simulation/sub8_sim_tools/rendering/visualization.py

```diff
@@ -48,34 +53,63 @@ def on_draw(self, event):
         self.world.draw(self.view)
 
     def on_key_press(self, event):
+        # Process events on a specific object
+        if(event.text.lower() == 'i'):
```

Comment:
Why make these ifs instead of elif? Only one key can be pressed at a time and intercepted